docs/topics/wasm/wasm-debugging.md

```diff
@@ -1,148 +1,115 @@
 application built with Kotlin/Wasm.
 
 ## Before you start
 
-Create a project using the Kotlin Multiplatform wizard:
-
-1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
-2. On the **New Project** tab, change the project name and ID to your preference. In this tutorial, we set the name to "WasmDemo" and the ID to "wasm.project.demo".
-
-   > These are the name and ID of the project directory. You can also leave them as they are.
-   >
-   {style="tip"}
+1. [Set up your environment for Kotlin Multiplatform development](https://www.jetbrains.com/help/kotlin-multiplatform-dev/quickstart.html#set-up-the-environment).
+2. Follow the instructions to [create a Kotlin Multiplatform project targeting Kotlin/Wasm](wasm-get-started.md#create-a-project).
 
-3. Select the **Web** option. Make sure that no other options are selected.
-4. Click the **Download** button and unpack the resulting archive.
+## Debug in IntelliJ IDEA
 
-   ![Kotlin Multiplatform wizard](wasm-compose-web-wizard.png){width=450}
+The Kotlin Multiplatform project you created contains a Compose Multiplatform application powered by Kotlin/Wasm.
+You can debug this application
+in IntelliJ IDEA out of the box, without additional configurations.
 
-## Open the project in IntelliJ IDEA
+> * Debugging Kotlin/Wasm code in IntelliJ IDEA is available starting from version 2025.3 of the IDE. If you created the
+> `WasmDemo` project in a different version of IntelliJ IDEA, switch to version 2025.3 and open the project there 
+> to continue this tutorial.
+> * To debug Kotlin/Wasm code in IntelliJ IDEA, you must have the JavaScript Debugger plugin installed. [See more information
+> about the plugin and how to install it.](https://www.jetbrains.com/help/idea/debugging-javascript-in-chrome.html#ws_js_debugging_chrome_before_you_start)
+>
+{style="note"}
 
-1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
-2. On the Welcome screen of IntelliJ IDEA, click **Open** or select **File | Open** in the menu bar.
-3. Navigate to the unpacked "WasmDemo" folder and click **Open**.
+1. In IntelliJ IDEA, open the Kotlin file to debug. In this tutorial, we'll work with the `Greeting.kt` file from the following directory
+   of the Kotlin Multiplatform project:
 
-## Run the application
+   `WasmDemo/composeApp/src/wasmJsMain/kotlin/wasm.project.demo.wasmdemo`
 
-1. In IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+2. Click on the line numbers to set breakpoints on the code that you want to inspect.
 
-   > You need at least Java 11 as your Gradle JVM for the tasks to load successfully.
-   >
-   {style="note"}
+   ![Set breakpoints](wasm-breakpoints-intellij.png){width=650}
 
-2. In **wasmdemo** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserDevelopmentRun** task.
+3. In the list of run configurations, select `wasmJsBrowserDevelopmentRun`.
```

Comment:
I think the name should be different. Please double-check it on your side.
<img width="301" height="159" alt="ij-run" src="https://github.com/user-attachments/assets/e1eef9d3-5c3e-468f-a3f7-bc5ff7568d2a" />